test/handler/test_phpstan_handler.vader

```diff
@@ -68,10 +66,57 @@ Execute(Output for .inc files should be parsed correctly):
   \   {
   \     'lnum': 9,
   \     'text': 'Access to an undefined property Test::$var.',
-  \     'type': 'W'
+  \     'type': 'E'
   \   }
   \ ],
   \ ale_linters#php#phpstan#Handle(bufnr(''), [
-  \   ' 1/1 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%',
   \   '/phpstan-test-files/foo/test.inc:9:Access to an undefined property Test::$var.',
   \])
+
+Execute(Test phpstan command output):
```

Comment:
Put these tests in a separate file in the `test/command_callback` directory. They may already be one. Use the special assertion commands documented in `:help ale-tests` instead.